frontend/src/presentation/pages/WorkspacePage.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from "react";
+import { Row, Col, Breadcrumb, Button } from "antd";
+import { QuestionList } from "../components/QuestionList";
+import { QuestionDetail } from "../components/QuestionDetail";
+import { Question } from "../../domain/entities/Question";
+import { questionUseCases } from "../../domain/usecases/QuestionUseCases";
+import styles from "./WorkspacePage.module.css";
+
+const WorkspacePage: React.FC = () => {
+	const [questions, setQuestions] = useState<Question[]>([]);
+	const [selectedQuestion, setSelectedQuestion] = useState<Question | null>(
+		null
+	);
+	const [isWorking, setIsWorking] = useState(false);
+	const [isLoading, setIsLoading] = useState(true);
+	const [error, setError] = useState<string | null>(null);
+
+	useEffect(() => {
+		const fetchQuestions = async () => {
+			try {
+				const fetchedQuestions = await questionUseCases.getAllQuestions();
+				setQuestions(fetchedQuestions);
+				setIsLoading(false);
+			} catch (error) {
+				console.error("Failed to fetch questions:", error);
+				setError("Failed to load questions. Please try again later.");
+				setIsLoading(false);
+			}
```

Comment:
Might be better to add `setIsLoading(false)` into `finally` block instead? This is fine too actually